pages/common/pnpm.md

```diff
@@ -0,0 +1,37 @@
+# pnpm
+
+> Fast, disk space efficient package manager for Javascript and Node.js.
```

Comment:
I don't like when package managers say "for JavaScript", because it's really just for Node.js
```suggestion
> Fast, disk space efficient package manager for Node.js.
```